example/android/app/build.gradle

```
@@ -24,8 +24,8 @@ if (flutterVersionName == null) {
 }
 
 android {
-    compileSdkVersion 32
-    ndkVersion flutter.ndkVersion
+    namespace "com.optimizely.optimizely_flutter_sdk_example"
+    compileSdkVersion 35
```

```suggestion
    compileSdkVersion 36
```
let's give it a test with different version in app